src/styles/buy.css

```diff
@@ -0,0 +1,19 @@
+.buy-link {
+  display: inline-block;
+  background: #00acdc;
+  border-radius: 5px;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  padding: 12px 69px 12px 70px;
+  margin-top: 16px;
```

Comment:
'buy-link' is your block. We can not give external styling like margin or positioning to block. You can create a element and then give it the external styling.